### PR TITLE
Adding Page for ml5 Object Detection Video

### PR DESCRIPTION
**Pull request by violetcraze**
Adding page for soon to be realeased ml5 video on object detection.

@shiffman below is what still needs to be done.

- [x] find or ask ml5 to publish website with object detection page
- [x] review description
- [x] update youtube description with any changes
- [x] add video to ml5 learning playlist on youtube

### CODE REVIEW

**Comment by shiffman** [comment]
@violetcraze thank you for this, awesome! For the ml5 object detection docs page in the YT description let's put a note that it is "forthcoming". It won't be live until the next release (but that should be within the next week or two.) Maybe we can do the same somehow on this page and remember to update it later?

File: _learning/ml5/9.1-object-detection.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+videos:
+  - title: "ml5.js Pose Estimation with PoseNet"
+    author: "The Coding Train"
+    url: "/learning/ml5/7.1-posenet"
+  - title: "Coding in the Cabana 4 - Worley Noise"
+    author: "The Coding Train"
+    url: "/challenges/coding-in-the-cabana/004-worley-noise"
+---
+
+In this video I check out the new object detection in ml5.js that uses the tensorflow.js coco-ssd model. Follow along to make a project that can detect 80 types of objects in an image or video!
```
Comment (shiffman) [inline _learning/ml5/9.1-object-detection.md:56]:
```suggestion
In this video, I cover object detection in ml5.js with the COCO-SSD pre-trained model.
```

File: learning/ml5/9.1-object-detection/2_webcam/index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <script src="https://cdn.jsdelivr.net/npm/p5@1.1.9/lib/p5.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/p5@1.1.9/lib/addons/p5.sound.min.js"></script>
```
Comment (shiffman) [inline learning/ml5/9.1-object-detection/2_webcam/index.html:6]:
```suggestion
```

File: learning/ml5/9.1-object-detection/1_image/index.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <script src="https://cdn.jsdelivr.net/npm/p5@1.1.9/lib/p5.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/p5@1.1.9/lib/addons/p5.sound.min.js"></script>
```
Comment (shiffman) [inline learning/ml5/9.1-object-detection/1_image/index.html:6]:
```suggestion
```

File: learning/ml5/9.1-object-detection/3_webcam_persistence/index.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <script src="https://cdn.jsdelivr.net/npm/p5@1.1.9/lib/p5.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/p5@1.1.9/lib/addons/p5.sound.min.js"></script>
```
Comment (shiffman) [inline learning/ml5/9.1-object-detection/3_webcam_persistence/index.html:6]:
```suggestion
```

Small thing but I think I prefer to remove p5.sound if it's not used by the example

File: _learning/ml5/9.1-object-detection.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    folder: "2_webcam"
+    lang: "p5js"
+    web_editor: VIYRpcME3
+  - name: "object detection webcam persistence"
+    folder: "3_webcam_persistence"
+    lang: "p5js"
+    web_editor: Vt9xeTxWJ
+
+links:
+  - title: "Object Detector ml5.js Reference"
```
Comment (shiffman) [inline _learning/ml5/9.1-object-detection.md:24]:
```suggestion
  - title: "Object Detector ml5.js Reference" (coming soon)
```

File: _learning/ml5/9.1-object-detection.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    url: 
+  - title: "COCO ml5.js Model and Data Provenance"
```
Comment (shiffman) [inline _learning/ml5/9.1-object-detection.md:26]:
```suggestion
  - title: "COCO ml5.js Model and Data Provenance" (coming soon)
```

**Review by shiffman — COMMENTED** [review]
Added some suggestion! Also, @violetcraze can you add the same header comments to the p5 web editor code as part of the process? Feel free to do any other cleanup to those examples too. Thank you!! 💖

**Comment by shiffman** [comment]
Any thoughts on where this video should go? I'm thinking of putting it in the "image classification" section?

Screen Shot 2020-10-17 at 7 32 52 PM

File: _learning/ml5/9.1-object-detection.md (hunk 5 above)
Comment (violetcraze) [reply _learning/ml5/9.1-object-detection.md:24]: Added link to the markdown ml5 page.

File: _learning/ml5/9.1-object-detection.md (hunk 6 above)
Comment (violetcraze) [reply _learning/ml5/9.1-object-detection.md:26]: Added link to the markdown ml5 page.

**Comment by violetcraze** [comment]
Merged via: https://github.com/CodingTrain/website/commit/671f5632f0979bce679c01d4ac3c292d52fd4ea3

**Pull request closed without merging**